### ISSUE TITLE
Inappropriate Required Node & NPM Version

### ISSUE THREAD

**Issue by ronny1982** [nwjs/npm-installer#109]
The latest changes introduce a forced minimum version of Node and NPM in the `package.json`.
This version is then used for the CI pipeline to configure the Node/NPM version.

Production code should not contain configurations for the automated test pipeline. Now all developers are forced to use a non-LTS Node/NPM version.

I suggest to either remove the required versions from the `package.json` and facilitate matrix builds in the CI pipeline with whatever Node/NPM versions, or require an appropriate (minimum) version in `package.json` and not the latest and greatest Node/NPM version. I'm pretty sure this package is even compatible when run with Node v16 LTS.

**Comment by ronny1982** [comment]
See also: #96

**Comment by ronny1982** [comment]
**Workaround:**
A project that uses `nw` must ensure not using `engine-strict = true`, so it only shows up as warning and not as error when installing the `nw` package dependency.

**Comment by ayushmanchhabra** [comment]
@ronny1982 Realised my mistake and fixed in #106. It should work with v0.81.x onwards. You can [patch the package](https://www.npmjs.com/package/patch-package) or unset `engine-strict`, as you've said above.

**Comment by tjmoore** [comment]
> @ronny1982 Realised my mistake and fixed in #106. It should work with v0.81.x onwards. You can [patch the current release](https://www.npmjs.com/package/patch-package) or unset `engine-strict`, as you've said above.

I'm using yarn and adding with `yarn add nw` and I get this with Node LTS, with 0.81.0

```
error nw@0.81.0: The engine "node" is incompatible with this module. Expected version "v20.5.0". Got "18.18.2"
error Found incompatible module.
```

and latest node 20 (using `nvm use 20`), still insists on specific version.

```
error nw@0.81.0: The engine "node" is incompatible with this module. Expected version "v20.5.0". Got "20.8.1"
error Found incompatible module.
```
